pytest_reporter/plugin.py

```diff
@@ -134,7 +134,7 @@ def pytest_report_collectionfinish(self, config, items):
         self.context["items"] = self._items
 
     def pytest_runtest_logstart(self, nodeid):
-        testrun = self._get_testrun(nodeid)
+        testrun = self._get_testrun(nodeid.split('@', 1)[0])
```

Comment:
I guess the _get_testrun() method does not need to be modified if the input has already been processed?

Reply:
If I don't change inside _get_testrun() method, its getting failed somewhere in Jinja2 lib.
I am new to Python, still analyzing why/where its failing.

If possible, Can you please help fixing this for the issue mentioned here,
https://github.com/christiansandberg/pytest-reporter-html1/issues/16

Reply:
@christiansandberg  - Can you please help on this.